sanic/exceptions.py

```diff
@@ -139,9 +139,12 @@ class PayloadTooLarge(SanicException):
 
 class Handler:
     handlers = None
+    cached_handlers = None
+    _missing = object()
```

Comment:
This is pedantic: but is there another variable name aside from `_missing` that makes sense here? For example, `_default`, `_base`, or `_empty_handler`?